libswirl/oslib/audiobackend_coreaudio.cpp

```diff
@@ -24,140 +24,127 @@
 #if HOST_OS == OS_DARWIN
 #include <atomic>
 
-//#include <CoreAudio/CoreAudio.h>
 #include <AudioUnit/AudioUnit.h>
 
-AudioUnit audioUnit;
-
-// ~ 93 ms
-#define BUFSIZE (4 * 1024 * 4)
-static u8 samples_temp[BUFSIZE];
+static AudioUnit audioUnit;
 
+// ~ 93 ms buffer
+#define SAMPLES_BUFSIZE (4 * 1024 * 4)
```

Comment:
Do you really need 4 megs of ring buffer here?

Reply:
@skmp thats in bytes, it’s 16KB ;)

Also that’s the original value, I didn’t change it I just changed the name.